### ISSUE TITLE
How to get queryset in sorted order?

### ISSUE THREAD

**Issue by NathanielHill** [gregmuellegger/django-sortedm2m#88]
I'm using DRF to expose my data via a REST API, and just realized I'm not serving it in the order it has been arranged in via the Django admin interface. What is the correct method for building the queryset in sorted order?

`queryset = GalleryPage.objects.all()`

**Comment by gregmuellegger** [comment]
So the `GalleryPage` has a sortedm2m field? And the data in there is not ordered when using with REST framework?

That shouldn't be the case. Does it work if you query GalleryPage directly, like
```
gallery = GalleryPage.objects.all().first()
gallery.photos.all() # is this ordered correctly?
```

**Comment by dutchbot** [comment]
I seem to have the same problem, tried what you suggested, but I get the following error:
```
AttributeError: 'SortedManyToManyDescriptor' object has no attribute 'all'

```

**Comment by NathanielHill** [comment]
FYI, I realized I was accessing the field from the wrong API endpoint. So, `queryset = Gallery.objects.all()` does not return a sorted list, but `queryset = GalleryPage.objects.all()` which contains the field `galleries = SortedManyToManyField(Gallery)` does work. So, the API spits my list of Gallery objects back in some unknown order (probably by time); but my GalleryPage object contains an ordered list of Gallery's. Would have been nice if that had auto-magically worked, but not sure what would be involved in that. Closing.
